Bio/KEGG/KGML/KGML_parser.py

```diff
@@ -80,7 +80,109 @@ def parse(handle, debug=0):
 
 
 class KGMLParser(object):
-    """Parses a KGML XML Pathway entry into a Pathway object."""
+    """Parses a KGML XML Pathway entry into a Pathway object.
+        Example: Read and parse large metabolism file
+
+    >>> from Bio.KEGG.KGML.KGML_parser import read
+    >>> pathway = read(open('KEGG/ko01100.xml', 'rU'))
+    >>> for k, v in list(pathway.entries.items())[:2]:
+    ...        print(v)
+    ...
+    Entry node ID: 1
+    Names: ko:K02821 ko:K02822 ko:K03475
+    Type: ortholog
+    Components: set()
+    Reactions: rn:R07671
+    Graphics elements: 1 ...
+    <BLANKLINE>
+    Entry node ID: 2
+    Names: ko:K03476
+    Type: ortholog
+    Components: set()
+    Reactions: rn:R07677
+    Graphics elements: 1 ...
+
+    >>> for r in list(pathway.reactions)[:2]:
+    ...     print(r)
+    ...
+    Reaction node ID: 1
+    Reaction KEGG IDs: rn:R07671
+    Type: reversible
+    Substrates: cpd:C00072
+    Products: cpd:C16186
+    <BLANKLINE>
+    Reaction node ID: 2
+    Reaction KEGG IDs: rn:R07677
+    Type: reversible
+    Substrates: cpd:C16186
+    Products: cpd:C14899
+    <BLANKLINE>
+
+    >>> print(len(pathway.maps))
+    149
+
+    >>> pathway = read(open('KEGG/ko00010.xml', 'rU'))
+    >>> print(pathway) #doctest: +NORMALIZE_WHITESPACE
+    Pathway: Glycolysis / Gluconeogenesis
+    KEGG ID: path:ko00010
+    Image file: http://www.kegg.jp/kegg/pathway/ko/ko00010.png
+    Organism: ko
+    Entries: 99
+    Entry types:
+        ortholog: 61
+        compound: 31
+        map: 7
+
+
```

Comment:
I will clean this up some. I would prefer to finish converting the self-tests to doc tests then create a new issue to improve the documentation. I don't really know anything about KGML/KEGG and would, therefore, need to learn about that before I have any chance to say anything intelligent about this.